mnswaleh/apisendIT
# bg-validate-inputs-161859453

#### What does this PR do?
Ensures all input required is entered

#### Description of Task to be completed?
should have methods to validate inputs
#### How should this be manually tested?
1 on postman, enter url to any put or get endpoint
2 omit relevant data and send request
3 should return appropriate response about the missing data

#### What are the relevant pivotal tracker stories?
#161859453
